components/CommentModal/CommentModal.js

```diff
@@ -60,35 +66,77 @@ My thoughts:
       ? "oppose"
       : "have thoughts on"
 
+  const emailSuffix = `See more testimony on this bill at ${webSiteBillAddress}${bill.BillNumber}`
+
+  const legislatorEmails =
+    representativeEmail && senatorEmail
+      ? representativeEmail + "," + senatorEmail
+      : representativeEmail
+      ? representativeEmail
+      : senatorEmail
+      ? senatorEmail
+      : null
+
   const mailIntroToLegislator = `As your constituent, I am writing to let you know I ${positionWord} ${bill?.BillNumber}: ${bill?.Title}.`
-  const mailToLegislators = encodeURI(
-    `mailto:${senatorEmail},${representativeEmail}?subject=${positionEmailSubject} Bill ${
-      bill ? bill.BillNumber : ""
-    }&body=${
-      testimony ? mailIntroToLegislator + "\n\n" + testimony.content : ""
-    }`
-  )
+
+  const mailToLegislators = !legislatorEmails
+    ? null
+    : encodeURI(
+        `mailto:${testimonyEmailAddress}?subject=${positionEmailSubject} Bill ${
```

Comment:
I don't follow your use of `testimonyEmailAddress`. Should this be to the legislator emails? Same for the committee email below

Reply:
Yes, I can see how this is not clear.  Matt asked if we could send the emails to legislators via BCC: instead of directly.  We can do that, but we need to have a primary address for the TO:
As a placeholder, I made up an email address called archive@digitaltestimony.com to be the primary address.  
There are [a number of domains like that available ](https://www.godaddy.com/domainsearch/find?domainToCheck=digitaltestimony.com&isc=GPPTCOM&key=parkweb&tmskey=dpp_dbs&utm_campaign=x_dom-broker_parkedpages_x_x_invest_001&utm_medium=parkedpages&utm_source=godaddy)if we want, but it could really be any email address.
One other advantage of this way we could also maintain an archive of emails that have been sent if we want.  
If people don't like this approach, we can change it.